keycloak/themes/uid2-theme/login/info.ftl

```diff
@@ -0,0 +1,25 @@
+<#import "template.ftl" as layout>
+<@layout.registrationLayout displayMessage=false; section>
+    <#if section = "header">
+        <#if messageHeader??>
+        ${messageHeader}
+        <#else>
+        ${message.summary}
+        </#if>
+    <#elseif section = "form">
+    <div id="kc-info-message">
+        <p class="instruction">${message.summary}<#if requiredActions??><#list requiredActions>: <b><#items as reqActionItem>${kcSanitize(msg("requiredAction.${reqActionItem}"))?no_esc}<#sep>, </#items></b></#list><#else></#if></p>
+        <#if skipLink??>
+        <#else>
+            <#if pageRedirectUri?has_content>
+              <script>location.href = '${pageRedirectUri}';</script>
+            <#elseif actionUri?has_content>
+              <p><a id="proceedWithAction" href="${actionUri}">${kcSanitize(msg("proceedWithAction"))?no_esc}</a></p>
+              <script>document.getElementById('proceedWithAction').click()</script>
```

Comment:
keycloak is setting cookie and proceeding action token at this page, using location.href will cause an infinity loop. So we have to use click instead.